pyEchosign/classes/account.py

```diff
@@ -32,6 +32,7 @@ def __init__(self, access_token, **kwargs):
         headers = {'Access-Token': access_token}
         response = requests.get(endpoints.BASE_URIS, headers=headers)
         response_body = response.json()
+        check_error(response)

```

Comment:
I would move this up a line, `check_error` also catches issues arising from using `.json()` on a response that doesn't actually contain json data.